src/modules/update-list.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable-next-line import/no-cycle */
+import { todoList, saveData } from './add-task.js';
+import { renderList } from './todolist.js';
+
+function checkTask(event) {
+  const { index } = event.target.dataset;
+  todoList[index].completed = event.target.checked;
+  saveData();
+}
+
+function clearCompleted() {
+  const incompleteTasks = todoList.filter((task) => !task.completed);
+  todoList.splice(0, todoList.length, ...incompleteTasks);
+
+  // update the index property of the remaining tasks
+  todoList.forEach((task, i) => {
+    task.index = i + 1;
+  });
+
+  saveData();
+  renderList();
+}
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Please refer to my first comment 👍

Reply:
ESLint was disabled for those parts of the code because terms the import as no named modules